Modules/Core/Source/Caching/Cache.swift

```diff
@@ -0,0 +1,23 @@
+public protocol CacheProtocol: AnyObject {
+    associatedtype Key
+    associatedtype Value
+
+    func get(key: Key) -> Value?
+
+    func set(value: Value, key: Key)
+}
+
+public class Cache<Key: AnyObject, Value: AnyObject>: CacheProtocol {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/MarcoSantarossa/mssample-ios/blob/576acfc7a9586a705531c2483c3f000240f3fdc0/Modules/Core/Source/Caching/Cache.swift#L10"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>